packages/swapkit/helpers/src/types/wallet.ts

```diff
@@ -22,6 +22,7 @@ export enum WalletOption {
   OKX = "OKX",
   OKX_MOBILE = "OKX_MOBILE",
   PHANTOM = "PHANTOM",
+  POLKADOT = "POLKADOT.JS",
```

Comment:
```suggestion
  POLKADOT_JS = "POLKADOT.JS",
```

I would be explicit about wallet name to avoid confusion with the chain

Reply:
Done